### PR TITLE
add photos to galleries and simple project gallery

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- add more photos in each gallery (service category)
- working in a simple project gallery

### CODE REVIEW

File: app/components/projects/gallery/ProjectsGallery.js [hunk 1]
```diff
@@ -1,80 +1,92 @@
...
 
-  function handleCategory1(event) {
-    setCategory1(event.target.value);
-
-    filterImages();
-  }
+export default function ProjectsGallery() {
+  return (
+    <div>
+      <div id={"complementar"}>
```
Comment (doublethink13) [inline app/components/projects/gallery/ProjectsGallery.js:13]: it seems all of these divs share the exact same structure

refactor them into a component please

File: app/components/projects/gallery/ProjectsGallery.js [hunk 2]
```diff
@@ -1,31 +1,52 @@
...
+        gallery={CONCEPT_BOARDS}
+        title={"Concept Boards"}
+        id={"concept"}
+      />
+    </div>
+  );
+}
 
-      <div id={"online"}>
+{
```
Comment (doublethink13) [inline app/components/projects/gallery/ProjectsGallery.js:48]: remove comments

File: app/components/services/galleries/concept_board.js [hunk 3]
```diff
@@ -43,24 +43,24 @@ const CONCEPT_BOARD = [
...
-    id: 5,
-    imgURL:
-      "https://chalet-webapp.s3.amazonaws.com/Servicos/decoracao/Sala+de+Jantar+Holanda/2.png",
-  },
-  {
-    id: 5,
-    imgURL:
-      "https://chalet-webapp.s3.amazonaws.com/Servicos/decoracao/Sala+de+Jantar+Holanda/3.png",
-  },
+  // {
```
Comment (doublethink13) [inline app/components/services/galleries/concept_board.js:46]: remove comments

File: app/components/services/galleries/decor.js [hunk 4]
```diff
@@ -29,39 +29,39 @@ const DECOR = [
...
   // },
-  {
-    id: 4,
-    title: "quarto",
-    category1: "consultoria",
-    category2: "sala",
-    imgURL:
-      "https://chalet-webapp.s3.amazonaws.com/Servicos/decoracao/20221026_193758.jpg",
-  },
+  // {
```
Comment (doublethink13) [inline app/components/services/galleries/decor.js:32]: comments

File: app/components/projects/gallery/ProjectsGallery.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/components/projects/gallery/ProjectsGallery.js]: done!

File: app/components/projects/gallery/ProjectsGallery.js (hunk 2 above)
Comment (Joanartdacosta) [reply app/components/projects/gallery/ProjectsGallery.js:48]: done!

File: app/components/services/galleries/concept_board.js (hunk 3 above)
Comment (Joanartdacosta) [reply app/components/services/galleries/concept_board.js]: done!

File: app/components/services/galleries/decor.js (hunk 4 above)
Comment (Joanartdacosta) [reply app/components/services/galleries/decor.js]: done!

File: app/components/projects/HouseMenu.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import COMPLEMENTARY from "../services/galleries/complementary";
+
+import projectStyles from "./projects.module.css";
+
+export default function HouseMenu() {
+  function FilterPlace() {
```
Comment (doublethink13) [inline app/components/projects/HouseMenu.js:6]: where is this used?

Comment (Joanartdacosta) [reply app/components/projects/HouseMenu.js]: you are right, deleted.

**Pull request merged**